include/clover_field.h

```diff
@@ -125,7 +124,12 @@ namespace quda {
     QudaCloverFieldOrder order;
     QudaFieldCreate create;
 
-    mutable double trlog[2];
+    mutable std::vector<double> trlog;
```

Comment:
Since this is fixed size, we could use an `std::array<double,2>` instead of a more general `std::vector<double>` here.

Reply:
Good suggestion.  d485e3030